docs/source/conf.py

```diff
@@ -46,7 +46,6 @@
     'sphinx.ext.autodoc',
     'sphinx.ext.autosummary',
     'sphinx.ext.doctest',
-    'sphinx.ext.intersphinx',
```

Comment:
We should also remove javaspinx from docs/requirements.txt . Are the java docs removal changes in master as well?

Reply:
I deleted the fork this PR was made from so I'll need to make a new PR with javasphinx change

Reply:
go for it